exercises/change/cases_test.go

```diff
@@ -0,0 +1,138 @@
+package change
+
+// Source: exercism/x-common
+// Commit: cda8f98 Create new exercises structure
```

Comment:
I got an error when running the example_gen.go file:

```
> go run example_gen.go 
2017/03/23 14:32:00 unexpected data structure: json: cannot unmarshal number into Go value of type []int
exit status 1
```
which I assume is from the update of the x-common file (https://github.com/exercism/x-common/commits/master/exercises/change/canonical-data.json)